DUI3-DX/Sdk/Speckle.Converters.Common/RootConvertManager.cs

```diff
@@ -27,7 +27,7 @@ public Base Convert(Type type, object obj)
 
       if (objectConverter == null)
       {
-        throw new NotSupportedException($"No conversion found for {type.Name}");
+        throw new NotSupportedException($"No conversion found for {type.Name} with actual type of {obj.GetType()}");
```

Comment:
Is it important to know the obj type here?

Reply:
This is for debugging purposes where the converter being asked for wasn't the type passed...this is a case for a lot of things at least in the Revit converter

Reply:
My only concern with this, is this is a very common user facing error message. And it would be nice for normal users to intuitively understand that "this type of object, isn't supported by us". I wasn't sure if this starts to confound some internal implementation complexity. 

Perhaps we should instead by wrapping higher up...
Not a blocker, I just expect we'll have some feedback about this change to the error message.